book/update-theme.py

```diff
@@ -13,13 +13,33 @@
 INDEX_HBS_PORT = 443
 INDEX_HBS_PROTO = "GET"
 INDEX_HBS_PATH = "/repos/rust-lang/mdBook/contents/src/theme/index.hbs"
+INDEX_HBS_DATA = None
 INDEX_HBS_HEADERS = {
     "user-agent": "Update index.hbs for +https://github.com/rinja-rs/rinja",
 }
 
+ETHICALADS_THEME = r'''
+<style>
+.ethical-sidebar {
+  margin-left: auto; }
```

Comment:
Also what happened with the indent here. XD

Reply:
Well, please fix it. :p

Reply:
Fixed. :)